### PR TITLE
fix(driver): fixed a couple of possible (yet unlikely) uninitialized vars in driver

### PR DESCRIPTION
**Pull request by FedeDP**
Issues spotted with `cppcheck`.

**What type of PR is this?**

/kind cleanup

**Any specific area of the project related to this PR?**

/area driver-bpf

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by FedeDP** [comment]
/hold
Not for the 0.32.1 release IMHO.

File: driver/bpf/fillers.h [hunk 1]
```diff
@@ -2367,6 +2367,7 @@ FILLER(proc_startupdate_3, true)
 	struct clone_args cl_args;
 #endif
 
+	res = PPM_FAILURE_BUG;
```
Comment (Andreagit97) [inline driver/bpf/fillers.h:2370]: Why here and not at line `2364`?

Comment (FedeDP) [reply driver/bpf/fillers.h:2370]: Yep in the bpf probe we can; on kmod we can't.
I respected the kmod c standard even if i hadn't to!

Comment (FedeDP) [reply driver/bpf/fillers.h:2370]: Well, above i didn't respect it ahah i will fix this.

**Review by jasondellaluce — APPROVED** [review]
LGTM

**Review by leogr — APPROVED** [review]

**Pull request merged**